src/mtx/commands/uplink/GetHalfHoursResponse.ts

```diff
@@ -0,0 +1,387 @@
+import Command, {TCommandExampleList, COMMAND_HEADER_SIZE} from '../../Command.js';
+import CommandBinaryBuffer, {IPeriod} from '../../CommandBinaryBuffer.js';
+import {READ_ONLY} from '../../constants/accessLevels.js';
+import {UPLINK} from '../../../constants/directions.js';
+import {IDate} from '../../utils/dateTime.js';
+import {TUint8} from '../../../types.js';
+
+
+interface IGetHalfHoursResponseParameters {
+    date: IDate,
+    periods: Array<IPeriod>,
+    /** if DST start/end of this day, contain DST hour */
+    dst?: TUint8
+}
+
+
+const COMMAND_ID = 0x15;
+const MIN_PERIODS = 48;
+const MIN_COMMAND_SIZE = 3 + (MIN_PERIODS * 2);
+const MAX_PERIODS = 50;
+const MAX_COMMAND_SIZE = 4 + (MAX_PERIODS * 2);
+
+const examples: TCommandExampleList = [
+    {
+        name: 'simple response',
```

Comment:
```suggestion
        name: 'response with 48 periods',
```
?

Reply:
simple - in case that it will be always 48 periods (48 half hours in each day except DST start/end)